packer/github-actions-runner/install-runner.sh

```diff
@@ -19,6 +19,11 @@ echo "Setting up GH Actions runner tool cache"
 sudo mkdir -p /opt/hostedtoolcache
 sudo chown -R ec2-user:ec2-user /opt/hostedtoolcache
 
+# The IDM server that AB2D needs to access doesn't support TLSv1.2 with EMS
+# See https://www.redhat.com/en/blog/tls-extended-master-secret-and-fips-rhel
+sudo update-crypto-policies --set FIPS:NO-ENFORCE-EMS
+sudo reboot
```

Comment:
Why a reboot in the middle of building the packer image? This will likely break the packer ssh session.